src/mame/layout/segaai.lay

```diff
@@ -41,15 +47,27 @@ license:CC0-1.0
 	<script><![CDATA[
 		file:set_resolve_tags_callback(
 				function ()
+					-- get overlay, if any
+					local overlay_image = file.device:memregion('cardslot:overlay')
+					file.elements['overlay']:set_draw_callback(
+							function (state, bitmap)
+								if overlay_image ~= nil then
+									-- TODO: reduce temporary memory usage when I/O classes are exposed to Lua
+									local data = overlay_image:read(0, overlay_image.size)
+									local image = emu.bitmap_argb32.load(data)
+									image:resample(bitmap)
+								end
+							end)
+
 					local touchpad = file.views['default'].items['touchpad']
 
 					-- recompute target touchpad cursor size and area when necessary
 					local curxoffs, curyoffs, curxscale, curyscale, curwidth, curheight
 					file.views['default']:set_recomputed_callback(
 							function ()
 								local lbounds = touchpad.bounds
-								curwidth = lbounds.width / 10
-								curheight = lbounds.height / 14
+								curwidth = lbounds.width / 16
+								curheight = lbounds.height / 16
```

Comment:
If you want the cursor to not be squashed, you need to consider the aspect ratio of the thing you’re referencing it to.  If you want the size to be based on the width divided by 16, you need to adjust the height factor by 4:2.8, or 10:7.  Something like `curheight = lbounds.height * 5 / 56` if I’m doing the mental arithmetic correctly.